template/page-dev-tools.html.jinja

```diff
@@ -0,0 +1,193 @@
+                    <p class="card-text">{{card.description}}</p>
+                  </div>
+                  <div class="card-footer">&nbsp;</div>
+                </a>
+              {% endfor %}
+              </div>
+          </div>
+          <div class="tab-pane" id="other" role="tabpanel" aria-labelledby="other-tab">
+            <h4>  {% trans %} Other {% endtrans %}</h4>
+            {% set other = [
+              {
+                "id": "domain",
+                "title": "Domain Verification Checker",
+                "description": "Verify your validator's domain.",
+                "href": "validator-domain-verifier.html",
+              },
+              {
+                "id":"xrp-ledger",
+                "title": "xrp-ledger.toml Checker",
+                "description":
+                  "Verify that your xrp-ledger.toml file is set up properly.",
+                "href": "xrp-ledger-toml-checker.html",
+              },
+              {
+                "id": "binary-visualizer",
+                "title": "Binary Visualizer",
+                "description":
+                  "Parse the XRP Ledger's native binary format with a visual representation breaking down the raw structure into its parts.",
+                "href": "https://richardah.github.io/xrpl-binary-visualizer/",
+              },
+              {
+                "id": "token-metadata-lookup",
+                "title": "Token Metadata Lookup",
+                "description":
+                  "Query known information about any token issued on the XRP Ledger.",
+                "href": "https://xrplmeta.org/",
+              },
+            ]%}
+            <div class="row row-cols-1 row-cols-lg-3 card-deck">
+            {% for card in other %}
+              <a class="card" href="{{card.href}}" target="_blank" id="{{card.id}}">
+                <div class="card-body">
+                  <h4 class="card-title h5">{{card.title}}</h4>
+                  <p class="card-text">{{card.description}}</p>
+                </div>
+                <div class="card-footer">&nbsp;</div>
+              </a>
+            {% endfor %}
+            </div>
+          </div>
+      </div>
+    </div>
+  </section>
+
+
+  <section class="container-new py-10 px-0">
+    <div class="col-lg-12 p-6-sm p-10-until-sm br-8 cta-card">
+      <img src="./img/backgrounds/cta-home-purple.svg" class="d-none-sm cta cta-top-left">
+      <img src="./img/backgrounds/cta-home-green.svg" class="cta cta-bottom-right">
+      <div class="z-index-1 position-relative">
+        <h2 class="h4 mb-8-sm mb-10-until-sm">{% trans %}Have an Idea For a Tool?{% endtrans %}</h2>
+        <p class="mb-10">{% trans %}Contribute to the XRP Ledger community by submitting your idea for a tool or open a pull request if you've developed a tool.{% endtrans %}</p>
+        <a class="btn btn-primary btn-arrow-out" href="https://github.com/XRPLF/xrpl-dev-portal/">Open a pull Request</a>
```

Comment:
Looks like you missed another translatable string. Also, Pull Request is capitalized inconsistently.

```suggestion
        <a class="btn btn-primary btn-arrow-out" href="https://github.com/XRPLF/xrpl-dev-portal/">{% trans %}Open a Pull Request{% endtrans %}</a>
```